.github/workflows/ci.yaml

```diff
@@ -49,15 +45,15 @@ jobs:
         sudo cp lib/*.a /usr/lib
         sudo ln -s /usr/lib/libgtest.a /usr/local/lib/libgtest.a
         sudo ln -s /usr/lib/libgtest_main.a /usr/local/lib/libgtest_main.a
-
     - name: Clone
       uses: actions/checkout@v3
-      with: 
+      with:
         submodules: true
     - name: Test
       run: |
         . ~/.spack/Spack/share/spack/setup-env.sh
-        export Kokkos_DIR=/home/runner/.spack/Spack/opt/spack/linux-ubuntu22.04-x86_64_v4/gcc-12.1.0/kokkos-3.7.01-zhffzchmugzhigeekgnatfeikkrjwmfw
+        # Find the Kokkos installation directory via spack & extract the path via grep & awk
```

Comment:
Thanks @faisal-bhuiyan. This is fairly high-level, so can you elaborate on the details? For example, I think something like this would be helpful to future readers:

```
`spack find -p` lists the package version and installation path
`grep -m 1 kokkos` extracts the line with the first instance of "kokkos"
`awk ...` extracts the second element of the grep and puts that into the env variable
```

The reason I'm pushing for this is that, while this works well, it's using a result from Spack that is probably not intended to be used to set an environment variable. That is to say that the developers of Spack may not consider the output of `spack find` to be a stable API. If this were parsing some environment variable set by Spack for this purpose, then I would consider it within the scope of the Spack documentation to instruct the usage. Here, though, if the Spack output changes format, this line will likely break and the reason may not be obvious to other OpenTurbine developers.

Reply:
Ah disregard, I just saw that you added similar information in the pull request description. My apologies, and thanks for putting it there!